chef/data_bags/crowbar/bc-template-provisioner.json

```diff
@@ -75,6 +75,7 @@
       "use_serial_console": false,
       "serial_tty": "ttyS0,115200n8",
       "dhcp": {
+        "for-unknown-nodes": true,
```

Comment:
Done in https://github.com/toabctl/barclamp-provisioner/commit/822fab6c191a4c559ebc2975a9a8f82403b57753